conf/arcc.config

```diff
@@ -0,0 +1,29 @@
+params {
+    config_profile_name = 'ARCC'
+    config_profile_description = 'Advanced Research Computing Center (ARCC) for the University of Wyoming'
+    config_profile_contact = 'Robert A. Petit III (@rpetit3)'
+    config_profile_url = 'http://www.uwyo.edu/arcc/'
+
+    // Custom params for ARCC
+    singularity_pull_docker_container = true
+
+    slurm_account = 'healthdatasci'
+    slurm_opts = ''
+    slurm_queue = 'teton,moran'
```

Comment:
Rather than specifying this with custom parameters, why not simple sub profiles within this config, so: `-profile arcc,teton`

Reply:
What happens then both partitions are to be used? 

Should I create a `teton_moran` profile along with individual profiles?

Reply:
here's an example: https://github.com/nf-core/configs/pull/622/files#diff-a45c821aa6dbf5cca764ee471f443619a08647e41cf112ef31b68831094ac821R178-R191

Reply:
I wonder if it would be ok to just have a default that uses the three partitions

```
-profile arcc
```

Then use the additional profiles to explicitily set the partition

```
-profile arcc,beartooth_bigmem
```

Haha I'm trying to avoid

```
-partition arcc,beartooth_moran_teton
```

Reply:
Sorry for spamming, but the reasoning for putting multiple partitions is to squeeze small jobs into available nodes with little wait time

Reply:
Yeah I think the profile example makes sense :+1: